### PR TITLE
full audit database backups

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 2222

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-546c6e85f9/how-to/)

File: content/cloud-databases/managing-backups-for-cloud-databases.md [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
 
--   The name of the backup is required, and the maximum length for the
-    name is 64 characters.
--   Backups are stored directly in your Cloud Files account and you will
-    be charged for the storage used.  Standard rates for Cloud Files
-    storage fees apply.
+4. Enter a name for the backup and a description, and then click
+   **Create Backup**
 
+   **Note:** The backup name is required cannot exceed 64 characters.
```
Comment (chri2547) [inline content/cloud-databases/managing-backups-for-cloud-databases.md]: ..and cannot exceed...

File: content/cloud-databases/managing-backups-for-cloud-databases.md [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
+last_modified_date: '2018-09-18'
+last_modified_by: Stephanie Fillmon
 product: Cloud Databases
 product_url: cloud-databases
 ---
 
-### Introduction
+Cloud Databases offers multiple ways to back up and restore your
+database. Backup files are stored in Cloud Files and are subject to
+standard Cloud Files charges. The files will remain in your
```
Comment (chri2547) [inline content/cloud-databases/managing-backups-for-cloud-databases.md]: remove 'will'

File: content/cloud-databases/managing-backups-for-cloud-databases.md [hunk 3]
```diff
@@ -55,58 +55,76 @@ during the backup process.
...
 
 ### Restore a Backup
 
-To restore a backup, click on the gear icon next to the backup name.
+To restore a backup to a new instance, use the following steps:
+
+1. Click the gear icon next to the name of the backup that you want to restore
+   and select **Restore**.
+
+   A diaglogue box displays.
```
Comment (chri2547) [inline content/cloud-databases/managing-backups-for-cloud-databases.md]: dialogue

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d7b4f8cc29/how-to/)

**Pull request merged**